code/script.js

```diff
@@ -1,51 +1,211 @@
+    <option value="In a hot air balloon">In a hot air balloon</option>
+  </select>
+    `;
+  } else if (travelplace === "The Korean Demilitarized Zone") {
+    inputWrapper.innerHTML = `
+    <select id="select">
+    <option value="" selected disabled>👇 Select means of travel below...</option>
+    <option value="Aboard a cargo ship">Aboard a cargo ship</option>
+    <option value="Human cannonball-style">Human cannonball-style</option>
+    <option value="By foot">By foot</option>
+    <option value="In a hot air balloon">In a hot air balloon</option>
+  </select>
+    `;
+  }
+  const select = document.getElementById("select");
+  select.addEventListener("change", () => nextQuestion(select.value));
+};
+
+const lifeInsurance = (waytotravel) => {
+  questionNumber++;
+  botReply(
+    `${waytotravel}? That's kinda weird but ok... have you updated your will and life insurance?`
+  );
+  inputWrapper.innerHTML = `
+    <button class="send-btn" id="yes">Yes</button>
+    <button class="send-btn" id="no">No</button>
+  `;
+  document.getElementById("yes").addEventListener("click", () => lastMessage());
+  document
+    .getElementById("no")
+    .addEventListener("click", () => noWilllifeinsurance());
+};
+
+const noWilllifeinsurance = () => {
+  botReply(
+    `You haven't? I strongly suggest you get your things in order before planning a trip with me. 😬`
+  );
+  inputWrapper.innerHTML = `
+  <button class="send-btn" id="restart">I understand</button>`;
+
+  document.getElementById("restart").addEventListener("click", () => {
+    location.reload();
+    return false;
+    buttonAudio.play();
+  });
+};
+
+const lastMessage = () => {
+  botReply(
+    `Then everything seems to be in order! That'll be four million euros. I only accept cash. "Safe" travels!`
+  );
+  inputWrapper.innerHTML = `
+  <button class="send-btn" id="restart">Thank you!</button>`;
+
+  document.getElementById("restart").addEventListener("click", () => {
+    location.reload();
+    return false;
+  });
+};
 
 // Set up your eventlisteners here
 
-// When website loaded, chatbot asks first question.
-// normally we would invoke a function like this:
-// greeting()
-// But if we want to add a little delay to it, we can wrap it in a setTimeout:
-// setTimeout(functionName, timeToWaitInMilliSeconds)
-// This means the greeting function will be called one second after the website is loaded.
+// This is an event listener that logs the name that the user writes in the input form at
+// the beginning of the convo, and then shows it in the user's reply bubble. It also gives the bot's
+// next question (with the user's name in it). The function for that is above.
+button.addEventListener("click", () => nextQuestion(input.value));
+input.addEventListener("keypress", (event) => {
+  if (event.key === "Enter" && input.value) nextQuestion(input.value);
+});
+
+//The first message from the bot will load 0,6 seconds after the page has loaded.
 setTimeout(greetUser, 1000);
```

Comment:
Really clear descriptive comments on this whole section, good job changing it to fit your code and make it make sense for you!